src/DynamoAnalyzer/AnalyzeEnvironment.cs

```diff
@@ -0,0 +1,18 @@
+using DynamoAnalyzer.Helper;
+
+namespace DynamoAnalyzer
+{
+    /// <summary>
+    /// Common dependencies used during the analysis
+    /// </summary>
+    internal static class AnalyzeEnvironment
```

Comment:
namespace and class name are too general.

Reply:
This class was renamed to WorkspaceHelper